downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -88,6 +88,12 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 ### Improvements
 
 #### OpenAPI Tool
+- Added mapping for the `additionalProperty` property in the OpenAPI object schema. With this support, generated Ballerina records can be open records or closed records according to the `additionalProperty` property details.
+  Thereby, some of your existing generated Ballerina records may change into close records after regenerating using Ballerina 2201.4.0 (Swan Lake) version.
+- Improved OpenAPI schema `nullable:true` property support by providing generated record field with default of value nil, instead of making field both nilable and optional.
+  Therefore, some of your existing generated Ballerina records field may change into default value nil using Ballerina 2201.4.0 (Swan Lake) version.
```

Comment:
```suggestion
  >**Info:** Therefore, the value of some of your existing generated Ballerina record fields may change into the default value (i.e., `nil`) when the Ballerina 2201.4.0 (Swan Lake) version is used.
```

Reply:
@lnash94 Do we really need to mention the impact for this case, as there won't be any breaking cases (correct me if I'm wrong) for the end users?

Reply:
Yeah, It won't be any breaking change. I added this in case the user gets confused with generated code after seeing the default nil assigning.
@aneeshafedo appreciate any thoughts regarding the connector behaviour ?

Reply:
Yes it won't be a breaking change for a connector user. Only when some connector is regenerated using the 2201.4.0 he will notice the changes in those records.